tests/tests.rs

```diff
@@ -200,6 +200,31 @@ fn build_entity() {
     assert_eq!(*world.get::<i32>(f).unwrap(), 789);
 }
 
+#[test]
+fn build_entity_clone() {
+    let mut world = World::new();
+    let mut entity = EntityBuilderClone::new();
+    entity.add("abc");
+    entity.add(123);
+    let e = world.spawn(&entity.build());
```

Comment:
What's the test coverage benefit of constructing two different builders?

Reply:
This follows the style of `built_entity` test. It may increase the coverage as one with many modifications and one with few modifications of already existing components. Will add testing for spawning multiple entities from the same builder